docker-compose.yml

```diff
@@ -1,15 +1,16 @@
 version: '2.1'
 
 services:
-  balena-cli-ui:
+  backend:
```

Comment:
This container also has the Quasar UI in it. Calling it backend might be a bit confusing?

Reply:
good thinking, I wasn't sure what to call it. maybe just `app` but that seems generic. Perhaps just `server` is explicit enough